cluster/aws/options.md

```diff
@@ -88,4 +88,26 @@ Defaults to vivid (Ubuntu Vivid Vervet), which has a modern kernel and does not
 Other options may require reboots, updates or configuration, and should be used only if you have a compelling
 requirement to do so.
 
+**NON_MASQUERADE_CIDR**
+
+The 'internal' IP range which Kuberenetes will use, which will therefore not
+use IP masquerade.  By default kubernetes runs an internal network for traffic
+between pods (and between pods and services), and by default this uses the
+`10.0.0.0/8` range.  However, this sometimes overlaps with a range that you may
+want to use; in particular the range cannot be used with EC2 ClassicLink.  You
+may also want to run kubernetes in an existing VPC where you have chosen a CIDR
+in the `10.0.0.0/8` range.
+
+Setting this flag allows you to change this internal network CIDR.  A sensible
+value is `100.64.0.0/10` - this is also a reserved range, and is much less
```

Comment:
AFAIK, yes (and it isn't like we're changing the default).  If a k8s user is using 100.64 at their office/home, I think it would be just like using 10/8 at your home; neither island can talk directly to the other.  But I honestly don't know - I could change the docs to be less of an endorsement of 100.64.  How about "Note that you must set other values consistently.  For example, if you chose 100.64.0.0/10, you could set: ...."